_issues/hawk.md

```diff
@@ -22,6 +22,14 @@ Representative Richard Hanna (R-NY):
 
+<<<<<<< HEAD
+* Sometimes military action is inevitable...and justified. For example, **Hillary supported the raid that killed Osama bin Laden**. However, she chastised GOP Presidential nominees like Ted Cruz for their reckless and cavalier talk, such as ["carpet bombing" ISIS](http://www.npr.org/2016/05/17/478361999/4-things-to-know-about-hillary-clintons-approach-to-foreign-policy). Propsals like carpet bombing don't sound tough, they sound like you don't have a real plan. Hillary knows that slogans are not a strategy, and loose cannons can be dangerous, which is why she has been a smart and steady leader in her role as Secretary of State. 
+
+* In regards to U.S. relations with Russia, Hillary has a strong record of exploring diplomatic options and attempts to strengthen relations. As Russia increased air force activity at the beginning of the Ukraine crisis,[Hillary was asked if she would shoot down a Russian jet](http://www.newsweek.com/where-do-clinton-and-trump-stand-russia-487777). She replied “That would not happen, because we're going to put up a no-fly zone where the Russians are clearly kept informed," Clinton said. "I want them at the table." In 2009, Hillary was a key architect of[the Russian reset](https://en.wikipedia.org/wiki/Russian_reset), aimed at strengthening U.S.- Russia relations under the Russian Foreign Minister Sergei Lavrov. The reset encouraged the reduction of both the U.S. and Russian nuclear arsenal. As Putin returned to power, Hillary urged President Obama to take a tougher line with the Russian President. She called for [stricter sanctions on Russia after Malaysia Airlines flight 17 was shot down](http://thehill.com/policy/international/212896-clinton-calls-for-tougher-sanctions-on-putin).
+
+* During the 3rd Democratic debate, Hillary made it clear that she would want [Russia included in a coalition to help destroy ISIS](https://www.washingtonpost.com/news/the-fix/wp/2015/12/19/3rd-democratic-debate-transcript-annotated-who-said-what-and-what-it-meant/). She has repeated her stance that “there is an important role for Russia to help in resolving the conflict in Syria.”
+=======
 * **Fact:** Sometimes military action is inevitable... and justified. For example, **Hillary supported the raid that killed Osama bin Laden**. However, she chastised GOP Presidential nominees like Ted Cruz for their reckless and cavalier talk, such as ["carpet bombing" ISIS](http://www.npr.org/2016/05/17/478361999/4-things-to-know-about-hillary-clintons-approach-to-foreign-policy):
 
 > Proposing that doesn't make you sound tough. It makes you sound like you're in over your head. Slogans aren't a strategy. Loose cannons tend to misfire. **What America needs is strong, smart, steady leadership to wage and win this struggle**.
```

Comment:
Use this, with the `> ` quote marker, instead of the first paragraph after `<<< HEAD`